src/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknow')
```

Comment:
- [ ] Since we are requested to strictly follow the requirements of this project and it says this class extends parent's constructor by adding @specialization as a parameter hence all  the other parameters in the parent's constructor should be present here plus @specialization. In this case, you're missing the parent_permission parameter :+1: I'm aware you surely felt as: "the teacher shouldn't obtain permission from the parent" but let's adhere to the requirements and grabs the important principle of inheritance in OOP :+1:

Reply:
@FrederickMih I was thinking exactly the same but not sure, Thanks for pointing it out, I'll make the correction now!